saleor/settings.py

```diff
@@ -305,9 +307,21 @@
             r'.+\.hot-update\.js',
             r'.+\.map']}}
 
+ENABLE_ELASTICSEARCH = ast.literal_eval(os.environ.get('ENABLE_ELASTICSEARCH', 'False'))
 
-BOOTSTRAP3 = {
-    'set_placeholder': False,
-    'success_css_class': '',
-
-}
+if ENABLE_ELASTICSEARCH:
+    HAYSTACK_CONNECTIONS = {
+        'default': {
+            'ENGINE': 'haystack.backends.elasticsearch_backend.ElasticsearchSearchEngine',
+            'URL': os.environ.get('ELASTICSEARCH_URL',
+                                  'http://127.0.0.1:9200/'),
+            'INDEX_NAME': os.environ.get('ELASTICSEARCH_INDEX_NAME', 'saleor')
```

Comment:
I think `storefront` would be a more reasonable default index name. I'd take descriptive database names over branded ones.